presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotQueryOptionsUtils.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.pinot;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class PinotQueryOptionsUtils
+{
+    private PinotQueryOptionsUtils()
+    {
+    }
+
+    public static boolean isNullHandlingEnabled(Map<String, String> queryOptions)
+    {
+        return Boolean.parseBoolean(queryOptions.get(PinotQueryOptionKey.ENABLE_NULL_HANDLING));
+    }
+
+    public static String getQueryOptionsAsString(Map<String, String> queryOptions)
+    {
+        if (queryOptions == null || queryOptions.isEmpty()) {
+            return "";
+        }
+        return queryOptions
+                .keySet()
+                .stream()
+                .map(key -> key + "=" + queryOptions.get(key))
```

Comment:
do we allow key or value contains quote? If so, do we need to escape here?

Reply:
It should not contains quotes but I am gonna add strip logic in case options were surrounded with quotes